pkg/alert/crdhandler.go

```diff
@@ -92,15 +90,6 @@ func (h *Handler) ObjectCreated(obj interface{}) {
 // ObjectDeleted will be called for delete alert events
 func (h *Handler) ObjectDeleted(name string) {
 	log.Debugf("objectDeleted: %+v", name)
-
-	apiClientset, err := clientset.NewForConfig(h.kubeConfig)
-	crd, err := apiClientset.ApiextensionsV1beta1().CustomResourceDefinitions().Get("alerts.synopsys.com", v1.GetOptions{})
```

Comment:
Why are we removing this verification?

Reply:
Namespace scope customer won't have access to CustomResourceDefinition permission.. CRD is at cluster scope...

Reply:
fixed